README.md

```diff
@@ -31,6 +31,10 @@ Install as usual, see [this](https://drupal.org/documentation/install/modules-th
 
 N/A
 
+## Documentation
+
+Further documentation for this module is available at [our wiki](https://wiki.duraspace.org/display/ISLANDORA/How+to+Batch+Ingest+Files)
+
```

Comment:
This is the closest thing we have in the user docs for Batch - there's very little in confluence for drush, period. Docs written by end-users for end-users. Leave it out? Link to the top level docs?